recipes/gromacs/2020/build.sh

```diff
@@ -41,46 +41,12 @@ ${CXX} -O3 -mavx512f -std=c++11 \
 -o ${PREFIX}/bin.AVX_512/identifyavx512fmaunits \
 ${SRC_DIR}/src/gromacs/hardware/identifyavx512fmaunits.cpp
 
+cp -a ${RECIPE_DIR}/gmx-chooser.bash ${PREFIX}/bin/gmx
+chmod a+x ${PREFIX}/bin/gmx
 
-# Create wrapper and activation scripts
-# Variable declaration from MPI script fewer changes if left in.
 
-gmx='gmx'
+# Copy the activate scripts to $PREFIX/etc/conda/activate.d
 
 mkdir -p "${PREFIX}/etc/conda/activate.d"
-touch "${PREFIX}/bin/${gmx}"
-chmod +x "${PREFIX}/bin/${gmx}"
-
-{ cat <<EOF
-#! /bin/sh
-
-function _gromacs_bin_dir() {
-  local arch
-  arch='SSE2'
-  case \$( grep -m1 '^flags' ) in
-    *\ avx512f\ *)
-      test -d "${PREFIX}/bin.AVX_512" && \
-        "${PREFIX}/bin.AVX_512/identifyavx512fmaunits" | grep -q '2' && \
-        arch='AVX_512'
-    ;;
-    *\ avx2\ *)
-      test -d "${PREFIX}/bin.AVX2_256" && \
-        arch='AVX2_256'
-    ;;
-    *\ avx\ *)
-      test -d "${PREFIX}/bin.AVX_256" && \
-        arch='AVX_256'
-  esac
-  printf '%s' "${PREFIX}/bin.\${arch}"
-}
-
-EOF
-} | tee "${PREFIX}/bin/${gmx}" > "${PREFIX}/etc/conda/activate.d/gromacs_activate.sh"
-
-cat >> "${PREFIX}/etc/conda/activate.d/gromacs_activate.sh" <<EOF
-. "\$( _gromacs_bin_dir )/GMXRC" "\${@}"
-EOF
-
-cat >> "${PREFIX}/bin/${gmx}" <<EOF
-exec "\$( _gromacs_bin_dir )/${gmx}" "\${@}"
-EOF
+cp ${RECIPE_DIR}/activate.sh "${PREFIX}/bin/gromacs_activate.sh" 
+ln -s  ../../../bin/gromacs_activate.sh "${PREFIX}/etc/conda/activate.d/gromacs_activate.sh"
```

Comment:
The relative directory with lots of `..` looks brittle, hard to understand, hard to maintain, or all three. Can it be expressed another way?

Reply:
Probably, but I am not sure what the directory structure is so will have to do some investigating.

Reply:
Thanks. I love getting rid of this bash magic, which I'd been dreading needing to ever understand!

Reply:
I have replaced it with ${PREFIX}, tests are passing, which is a good sign.

Reply:
That would have been my guess also